src/dotnet/project.json

```diff
@@ -26,6 +26,8 @@
     }
   },
   "dependencies": {
+    "NuGet.CommandLine.XPlat": "3.6.0-beta.1.msbuild.1",
```

Comment:
@piotrpMSFT just removed this reference and changed the non-msbuild based restore command to call NuGet in a separate process.  See https://github.com/dotnet/cli/pull/4024